src/ranking_watcher/usma.rs

```diff
@@ -12,9 +9,8 @@ use crate::scraping_utils::{first_nonempty_text, select_all, select_one, unpack_
 
 const RANKING_URL: &str = "https://ranking.cvgo.re/";
 
-type Result<T> = result::Result<T, Box<dyn Error + Send + Sync>>;
-
-#[derive(Debug, Eq, PartialEq)]
+#[derive(Eq, PartialEq)]
+#[cfg_attr(test, derive(Debug))]
```

Comment:
Any reason to only derive `Debug` for the test cfg? I believe it's always recommended to have `Debug` impl (https://rust-lang.github.io/api-guidelines/debuggability.html#all-public-types-implement-debug-c-debug), as effectively the only downside is slightly more code being generated (and possibly exposing some secrets, which is not the case here).